Automattic/newspack-plugin
# feat: replace WooCommerce’s login form with our own

### All Submissions:






### Changes proposed in this Pull Request:



Replaces the native WooCommerce login form with our own, utilizing the same markup as we use for the login modal (but with some additional styles to make it render inline).

To do so, ~we rely a bit on a hack to hijack the rendering of the WooCommerce login form template; when this template is about to be rendered, we replace with an empty template file and instead call our `render_auth_form` method to output the form markup. There may be a more elegant way to do this, but I couldn't find one in the short term (cc @claudiulodro if you have any suggestions).~ @miguelpeixe suggested a bit of a less-hacky way to do this by simply calling the `render_auth_form` method in the template itself.

Closes #1850.

### How to test the changes in this Pull Request:

1. Check out this branch.
2. In a new session, while not logged in, visit the My Account page to log in.
3. Confirm that you see our Reader Activation login form here instead of the usual native WC login form.
4. Confirm that all functionality works here as expected, and that once you log in successfully you're redirected to the My Account dashboard in the logged-in state.

### Other information:
